tutugodfrey/maintenance-tracker
# feat(resolve-request): admin is able to mark a request as resolved

#### What does this PR do?
create endpoint that allows an admin to resolve a request

#### Description of Task to be completed?
define put /requests/requestId/resolve 
create controller to handle the route

#### How should this be manually tested?
use postman to test the endpoint on https://mtrackers.herokuapp.com/api/v1/requests/requestId/resolve
ensure isAdmin is set to true
ensure requestId is pass req params



### What are the relevant pivotal tracker stories?
#157829154
